vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -298,6 +303,15 @@ def _get_accepted(
             torch.full((1, ), 1, device=target_probs.device))
         accepted = uniform_rand < capped_ratio
 
+        if os.environ.get('VLLM_MTP_PRINT_ACCPET_RATE', '0') != '0':
```

Comment:
How about change it to:
os.environ.get('VLLM_MTP_PRINT_ACCPET_RATE', '0').lower() in ('true', '1'):